src/ring_theory/jacobson.lean

```diff
@@ -474,7 +474,7 @@ begin
   { let ϕ : localization_map M (localization M) := localization.of M,
     rw ← (ϕ.map_comp _),
     refine ring_hom.is_integral_trans ϕ.to_map
-      (ϕ.map (M.mem_map_of_mem (φ : P'.quotient →* P.quotient)) ϕ') _ _,
+      (ϕ.map (submonoid.apply_coe_mem_map (φ : P'.quotient →* P.quotient)) ϕ') _ _,
```

Comment:
What's your preference for `M` being implicit vs implicit here @riccardobrasca? I'm happy either way, and only really care about consistency between the lemmas that share names. I'm fine with it as you have it now, but I'm also fine with you making `s`, `p` etc explicit in all the new lemmas.

Reply:
What about leaving the variable implicit in `mem_map_of_mem` (that seems the standard in mathlib), but explicit in `apply_coe_mem_map` to allow dot notation as it is currently used?

Reply:
Oh, there are a lot of `mem_map_of_mem`... I guess I will open another PR to add all the corresponding `apply_coe_mem_map` when we have decided which form is better.

Reply:
I agree that this can be postponed to a later PR.